### ISSUE TITLE
Xed 

### ISSUE THREAD

**Issue by nx-x** [linuxmint/xed#341]
nxx@wtf:~$ xed --version
xed - Version 2.2.3
nxx@wtf:~$ uname -a
Linux wtf 4.15.0-72-generic #81-Ubuntu SMP Tue Nov 26 12:20:02 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
nxx@wtf:~$ cat /etc/lsb-release 
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=19.2
DISTRIB_CODENAME=tina
DISTRIB_DESCRIPTION="Linux Mint 19.2 Tina"

Xed crash, syslog:

Dec 16 09:58:40 wtf xed[9575]: The program 'xed' received an X Window System error.#012This probably reflects a bug in the program.#012The error was 'BadLength (poly request too large or internal Xlib length erro'.#012  (Details: serial 4173170 error_code 16 request_code 18 (core protocol) minor_code 0)#012  (Note to programmers: normally, X errors are reported asynchronously;#012   that is, you will receive the error a while after causing it.#012   To debug your program, run it with the GDK_SYNCHRONIZE environment#012   variable to change this behavior. You can then get a meaningful#012   backtrace from your debugger if you break on the gdk_x_error() function.)
Dec 16 09:58:40 wtf kernel: [45362.992452] do_trap: 22 callbacks suppressed
Dec 16 09:58:40 wtf kernel: [45362.992454] traps: xed[9575] trap int3 ip:7efd93e3cea1 sp:7ffc70e26020 error:0 in libglib-2.0.so.0.5600.4[7efd93deb000+114000]

**Comment by collinss** [comment]
Please use the issue template - it's there for a reason. Random errors don't give any useful information that will allow us to fix the issue. We at least need a good back trace, and usually detailed, step-by-step instructions on how to reproduce the issue consistently before we can do anything about it. Please open a new issue and include all the information we ask for in the template, and we'll be happy to take a look. Also, please use a descriptive title for the issue like "x type error when I do y".
